utils/fhirpath/fhir_path_processor.bal

```diff
@@ -14,8 +14,9 @@
 // specific language governing permissions and limitations
 // under the License.
 
-import ballerina/lang.'int as langint;
 import ballerina/log;
+import ballerina/lang.'int as langint;
+import ballerina/lang.regexp;
 import ballerina/regex;
```

Comment:
```suggestion
import ballerina/lang.'int as langint;
import ballerina/lang.regexp;
import ballerina/log;
import ballerina/regex;
```

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerinax-health.fhir.r4/commit/67b875051877ed3079cd6ad622ba7ec426bddfef